src/main/java/com/launchacademy/giantleap/models/FashionItem.java

```diff
@@ -0,0 +1,80 @@
+package com.launchacademy.giantleap.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.URL;
+
+@Entity
+@Table(name="fashion_items")
+@Getter
+@Setter
+@NoArgsConstructor
+public class FashionItem {
+  @Id
+  @SequenceGenerator(name= "fashion_items_generator", sequenceName = "fashion_items_id_seq", allocationSize = 1)
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "fashion_items_generator")
+  @Column(name="id", nullable=false, unique=true)
+  private Integer id;
+
```

Comment:
currently these are all optional, from the model perspective. Is that intentional? 
If they are null false in the DB, ensure the model matches.

Reply:
Yes, it is intentional.